### ISSUE TITLE
PrepareForOutputStep before 99% completed

### ISSUE THREAD

**Issue by bilibilij** [ebi-pf-team/interproscan#194]
Hi,

We are using interproscan-5.50-84.0 .
We get many error lines after the 99％ completed line. 
Here are some of our error lines .
`23/03/2021 00:49:36:649 Welcome to InterProScan-5.50-84.0
23/03/2021 00:49:36:654 Running InterProScan v5 in STANDALONE mode... on Linux
23/03/2021 00:49:41:558 RunID: s4_20210323_004941407_j4et
23/03/2021 00:49:58:244 Loading file /export/home/longzq/jcf/2.annotation/1.interproscan/psze_pep_removeStop.fa
23/03/2021 00:49:58:247 Running the following analyses:
[CDD-3.18,Coils-2.2.1,Gene3D-4.2.0,Hamap-2020_05,MobiDBLite-2.0,PANTHER-15.0,Pfam-33.1,PIRSF-3.10,PRINTS-42.0,ProSitePatterns-2019_11,ProSiteProfiles-2019_11,SFLD-4,SMART-7.1,SUPERFAMILY-1.75,TIGRFAM-15.0]
Pre-calculated match lookup service DISABLED.  Please wait for match calculations to complete...
23/03/2021 00:50:16:001 Uploaded 51234 unique sequences for analysis
23/03/2021 07:20:52:872 25% completed
23/03/2021 08:20:53:548 42% completed
23/03/2021 09:07:56:240 75% completed
23/03/2021 09:34:21:207 90% completed
23/03/2021 10:34:34:336 99% completed
23/03/2021 11:34:44:893 99% completed
23/03/2021 12:34:58:893 99% completed
23/03/2021 13:35:07:403 99% completed
23/03/2021 14:35:23:230 99% completed
2021-03-23 14:58:55,519 [amqEmbeddedWorkerJmsContainer-16] [uk.ac.ebi.interpro.scan.management.model.implementations.PrepareForOutputStep:440] WARN - Proteing get: There is a problem processing results for p
2021-03-23 14:58:55,524 [amqEmbeddedWorkerJmsContainer-7] [uk.ac.ebi.interpro.scan.management.model.implementations.PrepareForOutputStep:440] WARN - Proteing get: There is a problem processing results for pr
2021-03-23 14:58:55,536 [amqEmbeddedWorkerJmsContainer-6] [uk.ac.ebi.interpro.scan.management.model.implementations.PrepareForOutputStep:440] WARN - Proteing get: There is a problem processing results for pr
2021-03-23 14:58:55,546 [amqEmbeddedWorkerJmsContainer-1] [uk.ac.ebi.interpro.scan.management.model.implementations.PrepareForOutputStep:440] WARN - Proteing get: There is a problem processing results for pr
2021-03-23 14:58:55,557 [amqEmbeddedWorkerJmsContainer-12] [uk.ac.ebi.interpro.scan.management.model.implementations.PrepareForOutputStep:440] WARN - Proteing get: There is a problem processing results for p
2021-03-23 14:58:55,567 [amqEmbeddedWorkerJmsContainer-19] [uk.ac.ebi.interpro.scan.management.model.implementations.PrepareForOutputStep:440] WARN - Proteing get: There is a problem processing results for p
2021-03-23 14:58:55,578 [amqEmbeddedWorkerJmsContainer-10] [uk.ac.ebi.interpro.scan.management.model.implementations.PrepareForOutputStep:440] WARN - Proteing get: There is a problem processing results for p
2021-03-23 14:58:55,588 [amqEmbeddedWorkerJmsContainer-14] [uk.ac.ebi.interpro.scan.management.model.implementations.PrepareForOutputStep:440] WARN - Proteing get: There is a problem processing results for p
2021-03-23 14:58:55,599 [amqEmbeddedWorkerJmsContainer-9] [uk.ac.ebi.interpro.scan.management.model.implementations.PrepareForOutputStep:440] WARN - `

There are 51,234 unique protein sequences for analysis. And our parameters for run are as followed.

`/export/databases/third_part/interproscan-5.50-84.0/interproscan.sh -i Z191_pep_removeStop.fa -d /export/home/longzq/jcf/2.annotation/1.interproscan  -goterms -iprlookup -pa -dp   `

It is my first time to use interproscan locally ,and I have ran the test_proteins.fasta passed before using the same parameters .
Is there anything wrong within my input protein sequences?

Thanks!

**Comment by gsn7** [comment]
this sometimes happens on some file systems and following the instructions below should resolve the problem:
```
cd <interproscan_install_dir>
wget ftp://ftp.ebi.ac.uk/pub/databases/interpro/iprscan/5/5.50-84.0/patch/interproscan-5.50-patch.zip
unzip -o  interproscan-5.50-patch.zip
```

as explained here:
https://interproscan-docs.readthedocs.io/en/latest/KnownIssues.html#error-after-more-than-90-completed

let us know if this works or not for you

**Comment by bilibilij** [comment]
Thanks , it works for me !

**Comment by gsn7** [comment]
this bug is now fixed in InterProScan-5.51-85.0
